data/uking_functions.csv

```diff
@@ -85887,26 +85887,26 @@
 0x000000710109b454,sub_710109B454,8,
 0x000000710109b45c,sub_710109B45C,208,
 0x000000710109b52c,sub_710109B52C,196,
-0x000000710109b764,nullsub_4475,4,
-0x000000710109b768,nullsub_4476,4,
-0x000000710109b76c,Bshop::parse,1404,
-0x000000710109bce8,sub_710109BCE8,872,
-0x000000710109c050,ResourceBshop::parse,28,
+0x000000710109b764,nullsub_4475,4,_ZN4ksys3res4Shop9doCreate_EPhjPN4sead4HeapE
+0x000000710109b768,nullsub_4476,4,_ZThn632_N4ksys3res4Shop9doCreate_EPhjPN4sead4HeapE
+0x000000710109b76c,Bshop::parse,1404,_ZN4ksys3res4Shop6parse_EPhmPN4sead4HeapE
+0x000000710109bce8,sub_710109BCE8,872,_ZN4ksys3res4Shop11parseTable_ERKi
+0x000000710109c050,ResourceBshop::parse,28,_ZThn632_N4ksys3res4Shop6parse_EPhmPN4sead4HeapE
 0x000000710109c06c,sub_710109C06C,588,
```

Comment:
Looks like this is also a Shop function (which is similar to Drop::findTableIndexOrNormal)